src/utils/scripts/backup.js

```diff
@@ -0,0 +1,59 @@
+const fs = require('fs-extra')
+const { join } = require('path')
+
+function checkErr (err) {
+  // NOTE: there is a bug where copySync will throw if the inode
+  // from the src and dst are equal. In the case of symlinks, that'll happen.
+  // https://github.com/jprichardson/node-fs-extra/issues/657
+  if (!err.toString().includes('Source and destination must not be the same')) {
+    throw err
+  }
+}
+
+function backup (userData, original) {
+  const bin = original.split('/').pop()
+  const backup = join(userData, bin + '.bak')
+
+  try {
+    fs.copySync(original, backup)
+  } catch (err) {
+    checkErr(err)
+  }
+
+  console.log(`${original} copied to ${backup}`)
```

Comment:
Any reason not to use `logger` here?

Reply:
As explained (not as well as it should) above:

> The **/scripts** directory are only to be used on this kind of external executed scripts. That's why I added them outside of ASAR.

So, imagine the *.js files in the `scripts` folder as... shell scripts. They're not linked to our Electron app since we're running Electron as Node. To use logger, we'd need access to Electron api.